Rename bundles to addons (#4981)
kyma-project/kyma: components/helm-broker/README.md

## Overview

> The Helm Broker is an implementation of a Service Broker which runs on the Kyma cluster and deploys applications into the Kubernetes cluster using Kyma addons, and the [Helm](https://github.com/kubernetes/helm) client. A addon is an abstraction layer over a Helm chart which allows you to represent it as a ClusterServiceClass in the Service Catalog. For example, a addon can provide plan definitions or binding details. The Helm Broker fetches addons definitions from an HTTP servers. A list of HTTP addons repositories is defined in the ConfigMap and can be changed in the runtime.

Reviewer: ```suggestion
The Helm Broker is a [Service Broker](https://kyma-project.io/docs/master/components/service-catalog/#service-brokers-overview) which exposes Helm charts as Service Classes in the [Service Catalog](https://kyma-project.io/docs/master/components/service-catalog/#overview-overview). To do so, the Helm Broker uses the concept of addons. An addon is an abstraction layer over a Helm chart which provides all information required to convert the chart into a Service Class.
```

> For the details about Helm Broker configuration, see [this](../../docs/helm-broker/03-04-fetch-addons-from-https-servers.md) document. See the [Create a addon](../../docs/helm-broker/03-01-create-addons.md) and [Bind addons](../../docs/helm-broker/03-02-bind-addons.md) documents to learn more about addons.

Reviewer: ```suggestion
For more information about the Helm Broker, read the [documentation](https://kyma-project.io/docs/master/components/helm-broker/).
```

> The Helm Broker implements the Service Broker API. For more information about the Service Brokers, see the [Service Brokers](../../docs/service-catalog/13-01-service-brokers.md) overview document.

Reviewer: ```suggestion

```
